packages/blitz-rpc/src/index-server.ts

```diff
@@ -174,14 +178,30 @@ export function rpcHandler(config: RpcConfig) {
       throw new Error("No default export for resolver path: " + routePath)
     }
 
+    let resolverConfig = (await loadableResolver()).config
+    if (!resolverConfig) {
+      resolverConfig = defaultConfig
+    }
```

Comment:
this doesn't take into account if user exports an empty object as the config. so we should ... spread in the defaults to an object, and let user provided config override the defaults